cloudinit/distros/bsd.py

```diff
@@ -127,3 +127,24 @@ def apply_locale(self, locale, out_fn=None):
 
     def apply_network_config_names(self, netconfig):
         LOG.debug('Cannot rename network interface.')
+
+    def shutdown_command(self, mode='poweroff', delay='now', message=None):
+        # called from cc_power_state_change.load_power_state
+
+        # direct copy from the generic shutdown() method, except that
+        # poweroff is with -p, not -P.
+
+        opt_map = {'halt': '-H', 'poweroff': '-p', 'reboot': '-r'}
```

Comment:
Can we turn that variable into a `Distro`  attribute ? We would them just need to update it here instead of redefining the whole `shutdown_command`  method

Reply:
done in ae8ee254 ; thanks for the suggestion